src/libs/actions/Session/index.js

```diff
@@ -29,23 +29,22 @@ Onyx.connect({
  * On Android, AuthScreens unmounts when the app is closed with the back button so we manage the
  * push subscription when the session changes here.
  */
-let previousAccountID;
+let previousNotificationID;
 Onyx.connect({
-    key: ONYXKEYS.SESSION,
-    callback: (session) => {
-        const accountID = lodashGet(session, 'accountID');
-        if (previousAccountID === accountID) {
+    key: ONYXKEYS.NVP_PRIVATE_PUSH_NOTIFICATION_ID,
+    callback: (notificationID) => {
+        if (previousNotificationID === notificationID) {
```

Comment:
You can get rid of this if check and `previousNotificationID` since it's only the notificationID changing in this Onyx key.

Reply:
Ah true! Changed